### ISSUE TITLE
Using label as ID

### ISSUE THREAD

**Issue by saantiaguilera** [emicklei/dot#16]
In DOT language, the ID is something apart from the label attribute. You are indexing the map by labels instead of by IDs.

This makes us unable to create subgraphs with a same label (but different IDs).

## Solution

Probably you should be indexing your `g.subgraphs` by `g.ID` instead of `g.Attr("label")`

**Comment by emicklei** [comment]
thank you for reporting this. I will have a look at the changes for this, hopefully it does not require a breaking one.

**Comment by saantiaguilera** [comment]
No worries. As a temporary fix I'm doing:
```go
s := graph.Subgraph(somethingUnique, dot.ClusterOption{})
s.Attr("label", "The same label I want for more subgraphs")
```

This way your map uses `somethingUnique` for iterating over the subgraphs but the DOT label used is "The same label I want for more subgraphs"

**Comment by emicklei** [comment]
The module has been updated with the fix, please use v0.10.2
